src/appsettings/settings.py

```diff
@@ -1210,3 +1210,58 @@ def check(self):
                 errors.extend(error.messages)
         if errors:
             raise ValidationError(errors)
+
+
+class NestedSetting(NestedDictSetting):
+    """Alias for NestedDictSetting."""
+
+
+class NestedListSetting(ListSetting):
+    """Nested list setting."""
+
+    def __init__(self, inner_setting, *args, **kwargs):
+        """
+        Initialization method.
+
+        Args:
+            inner_setting (Setting): setting that should be applied to list items.
+                NestedDictSetting is not supported at the moment.
+            name (str): the name of the setting.
+            default (dict): default value given to the setting.
+            required (bool): whether the setting is required or not.
+            prefix (str):
+                the setting's prefix (overrides ``AppSettings.Meta`` prefix).
+            call_default (bool): whether to call the default (if callable).
+            transform_default (bool): whether to transform the default value.
+            validators (list of callables): list of additional validators to use.
+            key_type: the type of the dict keys.
+            value_type (type): the type of dict values.
+            min_length (int): minimum length of the iterable (included).
+            max_length (int): maximum length of the iterable (included).
+            empty (bool): whether empty iterable is allowed. Deprecated in favor of min_length.
+        """
+        super(NestedListSetting, self).__init__(*args, **kwargs)
+        if not inner_setting.name:
+            inner_setting.name = self.name
+        self.inner_setting = inner_setting
+
+    def transform(self, value):
+        """Transform all list items."""
+        transformed_value = []
+        for item in value:
+            transformed_value.append(self.inner_setting.transform(item))
+        return transformed_value
+
+    def validate(self, value):
+        """Validate all list items."""
```

Comment:
I think we should override the `check` method, not the `validate`.

Reply:
That would make it impossible to create nested nested list settings (and also prevent using `NestedDictSetting` as inner setting in the future). You'd have to call `inner_setting.check()` inside that `check` method, but check method accesses `raw_value`, which tries to read the value from the django settings. This doesn't work for the `inner_setting`. Therefore I've decided to implement `validate` method instead.

Reply:
Yet this is what `NestedDictSetting` does currently.

If you look at the `raw_value` property, it does this:
```python
if self.parent_setting is not None:
    return self.parent_setting.raw_value[self.full_name]
else:
    return getattr(settings, self.full_name)
```

Couldn't you use the same trick (having a `parent_setting` attribute for each setting in the `NestedListSetting`)?

Reply:
That could be an answer, I'll try to make it work.

Reply:
OK, I updated the `raw_value` method of `Setting` and replaced `transform` and `validate` by `get_value` and `check`, which made `NestedListSetting` much more similar to `NestedDictSetting`. Unfortunatelly, it's still not possible to insert dict into list because of the default values.

Reply:
LGTM.